static/js/store-tab.js

```diff
@@ -1,18 +1,21 @@
 // This file is used to store the last active tab in the browser's local storage
 $(function() {
-    // when a tab is clicked
-    $('a[data-toggle="tab"]').on('shown.bs.tab', function (e) {
-      // save the latest tab using a cookie
-      localStorage.setItem('lastTab', $(this).attr('id'));
-    });
+  // generate a simple unique deterministic ID for the page
+  const pageid = window.location.pathname.replaceAll('/','_')
 
-    // get the last tab if it exists
-    var lastTabStories = localStorage.getItem('lastTab');
-    
-    if (lastTabStories) {
-      $('#' + lastTabStories).tab('show');
-    } else {
-      // Set the first tab to active if no active tab exists
-      $('a[data-toggle="tab"]:first').tab('show');
-    }
-  });
\ No newline at end of file
+  // when a tab is clicked
+  $('a[data-toggle="tab"]').on('shown.bs.tab', function (e) {
+    // save the latest tab using a cookie
+    localStorage.setItem('lastTab' + pageid, $(this).attr('id'));
+  });
```

Comment:
This is really nice, I wondered how it worked and that's a really nice touch. I wonder if that might also be useful for the trigger toggles/view experiences page? also flagging @helendduncan who I think had looked at that at some point to avoid overloadingt the URL with parameters?

Reply:
Yes, agreed! (Props to @mastoffel for coming up with the approach!).